src/proto/serializer.ts

```diff
@@ -60,6 +60,11 @@ export class ProtoSerializer {
             protoTransaction.GuardianSignature = transaction.guardianSignature;
         }
 
+        if (transaction.relayer && !transaction.relayer.isEmpty()) {
+            protoTransaction.Relayer = transaction.relayer.getPublicKey();
+            protoTransaction.RelayerSignature = transaction.relayerSignature;
+        }
```

Comment:
Let's extract to a function similar to the guardian check above, and also check if relayer signature is set.

Reply:
we discussed about this internally and said it is not needed